engine/runtime/src/main/scala/org/enso/compiler/SuggestionsSerializationManager.scala

```diff
@@ -0,0 +1,29 @@
+package org.enso.compiler
+
+import com.oracle.truffle.api.TruffleLogger
+import org.enso.pkg.Package
+import org.enso.polyglot.Suggestion
+
+import java.util.logging.Level
+
+final class SuggestionsSerializationManager(logger: TruffleLogger) {
```

Comment:
OK, so this is the functionality that needs to be filled. No surprise, I'd rather have these new classes in Java and not Scala.